### DISCUSSION TITLE
Pets Festival - Course de Pets

### DISCUSSION THREAD

**Discussion by Ravenclaw-l** [Crownicles/Crownicles#2556] [Nouvelles idées]
Cet évènement commence en août et se déroule sur deux semaines.
Foncez y participer car il ne se produit qu'une fois par an !
Au programme, deux courses totalement épiques de pets dressés, une chaque semaine.

**Récompenses et déroulement :**

- **La course** : chaque course se déroule sur une semaine entière, du lundi 00h00 UTC au dimanche 23h55 UTC.
Elle se compose de 10 étapes à franchir tout au long de la semaine.
Le parcours débute à la plage Sentinelle, traverse de nombreux lieux du royaume, et se termine par l'ascension épique du Mont Célestrum.
C’est la course mythique pour tout bon dresseur de pets (le nombre d'étoiles de votre pet ne change rien aux chances de victoire). Imaginez faire la course avec votre coq préféré et remporter cette course au nez et à la barbe des autres concurrents !

- **Gains pour les participants** : vous pourrez gagner de l'argent, des potions de temps, des items épiques ou légendaires aléatoires.

- **Badge spécial** : tous les participants qui terminent la course recevront un badge "Pets Festival".

- **Prix pour le vainqueur** : le gagnant de chaque course gagnera un objet mythique aléatoire ou un pet 5* exclusif (à voir avec le Staff).

**Mécanique de jeu :**

A chaque étape, vous serez confronté à 3 questions à choix multiple, chacune avec trois options de réponse.
La réponse choisie influencera le temps de trajet nécessaire pour franchir l'étape.
Bonne réponse : Temps de trajet entre 3h et 4h.
Réponse neutre : Temps de trajet entre 4h et 5h.
Mauvaise réponse : Temps de trajet entre 5h et 6h.

_exemple de questions :_

> Peut-on aller en prison ?
1/ Oui
2/ Non
3/ En volant des bananes

Le joueur choisit la réponse 3 (réponse neutre) et obtient +4h28 de trajet.

>Combien faut-il de points d'affection pour dresser un pet ?
1/ Aucun, des bons coups de tatane suffisent
2/ 50 points
3/ 100 points

Le joueur choisit la réponse 3 (bonne réponse) et obtient +3h34 de trajet.

>Quand sort la V5 ?
1/ À la rentrée
2/ En fin d'année
3/ La semaine des 4 jeudis

Le joueur choisit la réponse 3 (mauvaise réponse) et obtient +5h41 de trajet.

**Calcul du temps :**
Pour cette étape, le temps total de trajet est de 13h43.
Le joueur devra donc attendre ce cooldown pour franchir cette étape avant de débuter l'étape suivante.

**Objectif :**
Avec un peu d'organisation, il est possible de faire deux étapes par jour, en en faisant une le matin et une autre le soir.
Le pet du joueur qui atteint le Mont Célestrum en premier, au bout des 10 étapes avant la fin de la semaine, sera déclaré vainqueur !

>Allez-vous choisir le bon pet pour cette course mythique, et réussira-t-il à décrocher la récompense ultime : un objet Mythique ?
Vous le saurez en vous mesurant aux meilleurs dresseurs de pets du royaume !

![PETS FESTIVAL](https://github.com/user-attachments/assets/eaccae67-46a2-4d10-8308-df3a5a1070e7)

>Event réalisé et proposé en collaboration avec Procion

**Comment by QQtin3** [comment]
C'est une très bonne idée à mon sens, j'aime beaucoup le principe et un évent de ce genre devrait être adopté, cependant j'émets une certaine réticence concernant la récompense du vainqueur et une question sur l'attribution du badge.

Un item mythique représente actuellement dans le bot quelque chose comme 0.02% de drop et je doute très honnêtement que donner un item d'une telle rareté soit quelque chose de très réaliste, cependant le donc d'un objet d'une certaine rareté, aléatoirement est une bonne idée, c'est pourquoi je propose plus un item épique ou légendaire (dans la limite de la volonté de Draft) ce qui serait plus raisonnable.

Concernant le badge, serait-il uniquement donné au gagnant comme pour les tournois ? J'avoue avoir lu et je n'ai pas compris s'il y avait une subtilité ou non.


En attendant je reste largement favorable pour un tel ajout !

**Comment by Procion06** [comment]
merci pour ton com !
c'est juste une présentation pour vous soumettre cette idée, étant entendu que le tableau des récompenses est surtout là à titre d'exemple
ensuite ces récompenses sont ajustables selon ce que le Staff juge acceptable ou non
donner un badge à tous les participants qui ont terminé la course (les 10 étapes) nous semblait cohérent mais là encore ça reste une suggestion

nous proposons un event annuel festival, à introduire durant l'été donc, dans le but de regrouper un maximum de monde pendant cette période
mais rien n'empêche d'en faire un event mensuel en limitant à une seule course dans le mois, sur le même principe que l'île mais tous les mois au lieu de chaque semaine

Raven et moi avons pris plaisir à vous proposer cette idée, en pensant que ça pourrait apporter un peu de fun au joueurs, libre à vous ensuite de la remodeler selon vos envies, si au final ça débouche sur une maj, quelle qu'elle soit, nous en serons ravis :)

**Comment by BananePlantain** [comment]
je l'avais posté dans rédaction je le met ici aussi
Bonne idée ! Il faudrait voir à modifier les issues de la question parceque se prendre 3h supplémentaire avec la BONNE réponse me semble délirant/punitif et avoir un peu trop d'impact sur les temps de trajets .Je propose de mettre à la place soit
A)bonne réponse= -1h de trajet ;réponse neutre =temps de trajet inchangé ;mauvaise réponse ;+1h de trajet
B)bonne réponse= proba plus faible d'avoir des altés/proba plus forte d'avoir des gains de temps lors du trajet ;réponse neutre : proba gain de temps/altés inchangés ; mauvaise réponse=proba plus forte d'avoir des altés/proba plus faible d'avoir des gains de temps
Es ce que les questions remplace les évenements ? dans ce cas il faudrait un /participer pour que tu sois pas forcé à répondre à des questions sans le vouloir

**Reply by Ravenclaw-l** [reply]
Merci pour ton retour et désolé pour le temps de réponse. 
Réduire le temps de trajet de 3h à 1h (en cas de bonnes réponses) rendrait le jeu trop facile et trop court. Le but, c'est que l'événement se passe sur une semaine. Si on raccourcit trop, ça risque de devenir trop rapide à terminer.
Concernant les questions, une fois que le joueur reprend sa course avec /festival ou /coursedepets, les questions peuvent s'enchaîner automatiquement : dès qu'une est répondue, la suivante s'affiche. Il faudra donc que le joueur soit bien focus sur les questions avant de lancer sa commande.

**Reply by Ravenclaw-l** [reply]
/festival ou /coursedefamiliers.
parce que /coursedepets pourrait faire penser à un festival complètement différent.
